fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -2,8 +2,10 @@ import Ember from 'ember';
 import request from 'ic-ajax';
 
 export default Ember.Route.extend({
-  model() {
-    return this.store.queryRecord('deployment', {id: this.modelFor('deployment').get('id'), sync_openstack: true});
```

Comment:
Yes, before it was taking the customized deployment which took in query parameters.  I didn't need to use the sync_openstack param and it was deleted in deployment.js